src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
The message "Truck started its work" should be more informative. It could indicate what type of machine, in this case, 'Truck', is working right now. For example: "Truck is now working on transporting goods."